src/styles/respec.css.js

```diff
@@ -218,34 +218,45 @@ aside.example .marker > a.self-link {
   color: inherit;
 }
 
-:is(h2, h3, h4, h5, h6) + a.self-link {
-  border: none;
+.header-wrapper {
+  display: flex;
+  align-items: center;
+}
+
+:is(h2, h3, h4, h5, h6):not(#toc h2) {
+  position: relative;
+  left: -.5em;
+  border-bottom: 1px solid #ddd;
```

Comment:
Why adding this border?

Reply:
Debugging accident... I thought I'd killed it...